app/views/provider/admin/cms/_includes.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= javascript_include_tag 'provider/cms' %>
-<%= javascript_include_tag 'provider/codemirror' %>
+<%= javascript_pack_tag 'codemirror' %>
```

Comment:
I don't like the idea of this "incudes" html. Include the pack at the uppermost possible level.